tests/Integration/DataAccess/SerializedDataHandlingTest.php

```diff
@@ -20,7 +23,13 @@ class SerializedDataHandlingTest extends TestCase {
 
 	private const MEMBERSHIP_APPLICATION_ID = 715;
 
-	/** @dataProvider encodedMembershipDataProvider */
+	/** @dataProvider encodedMembershipDataProvider
+	 * @param array<string, mixed> $data
+	 *
+	 * @throws ORMException
+	 * @throws OptimisticLockException
+	 * @throws Exception
```

Comment:
We need to talk about annotating the methods for these exceptions

Reply:
Done.